back/api/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from dataclasses import dataclass
+import os
+import subprocess
+
+# Create your models here.
+@dataclass
+class Student:
+    username: models.CharField(null=True, blank=True, max_length = 255)
+    updated: models.DateTimeField(auto_now=True)
+    created: models.DateTimeField(auto_now_add=True)
+
+    def __str__(self) -> str:
+        return self.username[0:50]
+
+class Exercise(models.Model):
+    """ Exercise composed of the statement (in markdown) and the solution 
+    (in Python) written by the teacher. The test_input set  (defined by 
+    the teacher) and the correct_output which is the output expected after 
+    execution of the correct solution on test_input."""
+    statement = models.TextField(null=True)
+    solution = models.TextField(null=True)
+    test_input = models.CharField(null=True, max_length = 255)
```

Comment:
I don't understand why the test's inputs are stored in a charfield and not a Textfield as the other attributes of an exercise?